app/Main.hs

```diff
@@ -68,6 +68,7 @@ main = do
             [ "An entry already exists at '" <> build path <> "'."
```

Comment:
Please rebase this branch on top of `diogo/#23-bats-tests`, and then write tests for this PR.

Reply:
Done